app.py

```diff
@@ -52,25 +53,26 @@ def confirm_login():
 def register():
    return render_template("register.html")
 
-
 @app.route("/register/resultsjson", methods = ['POST','GET'])
 def resultsJSON():
    if request.method == 'POST':
       results_data = request.form
       # converting dictionary to json
-      data_json = json.dumps(results_data)
+      data_json = json.dumps([results_data])
       # read to register.json file
       with open("register.json","r") as outfile:
          # creating a json object in the form of key/value pair
          account = json.load(outfile)
-         if("email" in account == data_json.get("email")):
+         shared_items = {k: account[k] for k in account if k in data_json and account[k] == data_json[k]}
```

Comment:
What happens if they share the same user but the data within the saved accounts is different ?

Example:

`account[k] = {"username": "bob", "psw": "some_hash"}`
`data_json[k] = {"username": "bob", "email": "something", "data": "something"}`

Reply:
if they share the same data it should display user already exists... I'm looping inside the 2 dictionaries and checking the values inside the 2 dictionaries....Its comparing btwn the already saved data(data_json[k])  to the current inputed data (account[k])